src/main/java/Duke.java

```diff
@@ -1,11 +1,39 @@
 import java.util.Scanner;
 public class Duke {
 
-    public static void printDivider() {
+    public static final int TASK_LIST_SIZE = 100;
+
+    public static final String COMMAND_BYE = "bye";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_MARK = "mark";
+    public static final String COMMAND_UNMARK = "unmark";
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_DEADLINE_BY = " /by ";
+    public static final String COMMAND_EVENT = "event";
+    public static final String COMMAND_EVENT_FROM = " /from ";
+    public static final String COMMAND_EVENT_TO = " /to ";
```

Comment:
Perhaps you could put all your constants into a separate class? It could make your code neater.